### PR TITLE
added JavaScript Question 2

### PR DESCRIPTION
**Pull request by dutta-pratik**
Please review and merge

### CODE REVIEW

File: javascript/answers/anonymous.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                <h4>
+                    How to define an anonymous function?
+                </h4>
+                
+            </div>
+            
+            
+            <div>
+                <strong>Answer:</strong> 
+                <p>An anonymous function can be defined in a similar way as a normal function but it would not have any name.</p>
```
Comment (shivamyth) [inline javascript/answers/anonymous.html]: Example of an anonymous function would make this better.

**Review by shivamyth — CHANGES_REQUESTED** [review]

File: javascript/answers/anonymous.html (hunk 1 above)
Comment (dutta-pratik) [reply javascript/answers/anonymous.html]: done!

File: javascript/index.html [hunk 2]
```diff
@@ -111,7 +111,9 @@ <h4>
             <li>
                 <a href="./answers/q5.html"><h4>What are escape Sequences?</h4></a>
             </li>
-
+            <li>
+                <a href="./answers/anonymous.html"><h4>How to define an anonymous function?</h4></a>
```
Comment (shivamyth) [inline javascript/index.html]: name of answer file is in the wrong format, correct it

**Review by shivamyth — DISMISSED** [review]

File: javascript/index.html (hunk 2 above)
Comment (dutta-pratik) [reply javascript/index.html]: done!

**Review by dakshkhetan — DISMISSED** [review]

**Review by dakshkhetan — APPROVED** [review]

**Pull request merged**